test/acceptance/behave/components/common_steps/general_steps.py

```diff
@@ -163,6 +163,17 @@ def check_in_log_label_and_text(context, label, text):
     __logger__.info("...confirmed traces in log")
 
 
+@step(u'delay for "([^"]*)" seconds')
+def delay_for_seconds(context, seconds):
+    """
+    delay for N seconds
+    :param
```

Comment:
Missing blank like between function description and parameters, and between last `"""` and code (I think that PEP8 will complain about that last one at least).

`seconds` is not included in param descriptions.